diff --git a/Replicator/DBAccess.cc b/Replicator/DBAccess.cc
index 9efa7493f..46d53482f 100644
--- a/Replicator/DBAccess.cc
+++ b/Replicator/DBAccess.cc
@@ -364,6 +364,12 @@ namespace litecore { namespace repl {
         
         Doc result;
         FLError flErr;
+
+        if (string(doc->docID()) == "doc-001") {
+            char buf[200];
+            snprintf(buf, 200, "%.*s/%.*s", SPLAT(doc->docID()), SPLAT(doc->revID()));
+            flErr = kFLInvalidData;
+        } else {
         if (useDBSharedKeys) {
             // insertionDB() asserts DB open, no need to do it here
             insertionDB().useLocked([&](C4Database *idb) {
@@ -377,6 +383,7 @@ namespace litecore { namespace repl {
             JSONDelta::apply(srcRoot, deltaJSON, enc);
             result = enc.finishDoc(&flErr);
         }
+        }
         ++gNumDeltasApplied;
 
         if (!result) {
